samples/gibbs_ensemble/gibbs_ensemble_socket.py

```diff
@@ -53,7 +53,7 @@
 parser.add_argument('-C', '--client-script', nargs=1)
 parser.add_argument('-n', '--number-of-particles', type=int, nargs=1)
 parser.add_argument('-l', '--box-length', type=float, nargs=1)
-parser.add_argument('-T', '--temperature', type=float, nargs=1)
+parser.add_argument('-T', '--temperature', type=float, nargs=1, required=True)
```

Comment:
it's kind of odd to declare an optional argument and make it required, why not use a positional argument?